x/mongo/driver/compression_test.go

```diff
@@ -27,25 +24,19 @@ func TestCompression(t *testing.T) {
 
 	for _, compressor := range compressors {
 		t.Run(strconv.Itoa(int(compressor)), func(t *testing.T) {
-			payload := "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt"
+			payload := []byte("Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt")
 			opts := CompressionOpts{
 				Compressor:       compressor,
 				ZlibLevel:        wiremessage.DefaultZlibLevel,
 				ZstdLevel:        wiremessage.DefaultZstdLevel,
 				UncompressedSize: int32(len(payload)),
 			}
-			compressed, err := CompressPlayoad([]byte(payload), opts)
-			if err != nil {
-				require.Contains(t, err.Error(), "support requires cgo")
-				return
-			}
+			compressed, err := CompressPlayoad(payload, opts)
```

Comment:
CompressPlayoad -> CompressPayload.  This is in x so its ok to change.

Reply:
Oops, fixed